### ISSUE TITLE
Make resource type footer sit at the bottom of the card

### ISSUE THREAD

**Issue by pmahnke** [canonical-websites/insights.ubuntu.com#103]
Would be neater to have the resource 'type' label sit attached to foot of card. See screenshot.

I'll let one of the visual designers have a deeper look though. 
![ubuntu insights - suggestion 1](https://user-images.githubusercontent.com/17002826/34885085-b2508f80-f7b6-11e7-80a9-445daff8e7d1.png)

**Comment by anthonydillon** [comment]
Equal heights flex only effects is direct children. So position: absolute
should work here.

**Comment by frankiegranato** [comment]
@Caleb-Ellis  I think we should pause this work because Spencer isn't happy with the overall design yet. See email below for more detail:

> Spencer Bygraves to me 9:49 AM: 
Hi Frankie,

Hope you had a good weekend.

After seeing the insights work on Friday, I'm not sure that the coloured bars at the top of each is card is the best thing to do for resources. When I saw the orange with white text I realised I hadn't considered accessibility. I don't think any of those secondary colours have enough contrast to use white text, I'll check.

So, I'm going to try a coloured strip with dark text or similar. I spoke to Caleb just now, he's cool if it changes a bit.

I have a workshop all day, will squeeze it in when I can and send you something to see what you think.

Thanks
